networking/using-ptp.adoc

```diff
@@ -5,13 +5,18 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: Precision Time Protocol (PTP) hardware
+:FeatureName: Precision Time Protocol (PTP) hardware with dual NIC and boundary clock
 include::modules/technology-preview.adoc[leveloffset=+1]
 
+[TIP]
```

Comment:
As with #39734, I think that the level of importance for this admonition is a note according the SSG, and there are a lot of admonitions clustered together. Is there a way to combine the notes into one call out, and break up the placement without losing meaning? 

I recognize that our options are somewhat limited by using the TP module.

Reply:
One option might be to manually combine the two as a single `IMPORTANT` admonition (borrowing the text from the TP module). Also, it would be nice to have an intro sentence or two for the top of this assembly, but I realize that's not in the scope of this PR, so maybe something for a future PR.

Reply:
Thanks both. I think there is sill some conversation ongoing regarding the status of these features TBC...